src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,64 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static final String DEFAULT_DELIMITER = ",|:";
+    public static final String CUSTOM_DELIMITER = "//(.*?)\n(.*)";
+
+    public static int splitAndSum(String text) {
+        if (isNullOrBlank(text)) {
+            return 0;
+        }
+
+        return sum(toInt(split(text)));
+    }
+
+    private static int sum(int[] numbers) {
+        int sum = 0;
+
+        for (int number : numbers) {
+            sum += number;
+        }
+
+        return sum;
+    }
+
+    private static String[] split(String text) {
+        Matcher m = Pattern.compile(CUSTOM_DELIMITER).matcher(text);
```

Comment:
Pattern 객체도 매번 생성하기 보다 상수로 선언해 두면 어떨까요?
```suggestion
        Matcher m = CUSTOM_DELIMITER_PATTERN.matcher(text);
```